packages/arb-token-bridge-ui/src/components/TransactionHistory/helpers.ts

```diff
@@ -386,31 +386,35 @@ export async function getUpdatedEthDeposit(
     return { ...tx, status: 'failure', depositStatus: DepositStatus.L1_FAILURE }
   }
 
-  const status = await parentToChildMsg?.status()
```

Comment:
All of the previous code was just wrapped inside `if (isEthDepositMessage(parentToChildMsg))`. In the other case, we just call `getUpdatedRetryableDeposit`.